docs/static/settings-file.asciidoc

```diff
@@ -28,4 +29,82 @@ pipeline.batch.size: 125
 pipeline.batch.delay: 5
 -------------------------------------------------------------------------------------
 
-See <<command-line-flags>> for a description of the available options.
+The `logstash.yml` file includes the following settings:
+
+ *`node.name`*::
```

Comment:
@suyograo I think you're saying that you would like to see these options presented in a 3 column table instead of a definition list, right? In that case, I would suggest that we have the column headings Setting, Description, and Default value. For this particular content, I OK with formatting the info as a table (though I think the descriptions are a tad long). My only caveat is that we shouldn't use a table format if we expect that we'll ever need to add multi-line code examples to a description. Multi-line code examples in tables of 3 or more columns don't display very well because of how line wrapping works. Definition lists are much more flexible, which is why we tend to use them for reference content. So...I'm open to reformatting. Let me know if you still want me to do this.